crates/providers/blobstore-fs/src/lib.rs

```diff
@@ -1 +1,750 @@
-pub mod fs_utils;
+//! blobstore-fs capability provider
+//!
+//!
+
+use std::time::SystemTime;
+use std::{
+    collections::HashMap,
+    io::{Error as IoError, ErrorKind as IoErrorKind},
+    path::{Path, PathBuf},
+    sync::Arc,
+};
+
+use async_trait::async_trait;
+use path_clean::PathClean;
+use serde::Deserialize;
+use tokio::fs::{
+    create_dir_all, metadata, read, read_dir, remove_dir_all, remove_file, File, OpenOptions,
+};
+use tokio::io::AsyncWriteExt;
+use tokio::sync::RwLock;
+use tracing::{error, info};
+use wasmcloud_provider_sdk::{
+    core::LinkDefinition,
+    error::{ProviderInvocationError, ProviderInvocationResult},
+    Context,
+};
+
+mod fs_utils;
+use fs_utils::all_dirs;
+
+wasmcloud_provider_wit_bindgen::generate!({
+    impl_struct: FsProvider,
+    contract: "wasmcloud:blobstore",
+    wit_bindgen_cfg: "provider-blobstore"
```

Comment:
what does this do?

Reply:
The value on this line is actually passed through to the bindgen *underneath* `wasmcloud_provider_wit_bindgen` -- i.e. passed to the upstream code, so that the same configurability is available to the person calling generate.

Not all the forms are well documented, but it's using the form that is equivalent to the world name only, without any other special options being passed.